### DISCUSSION TITLE
Croatian Language Leader

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#365] [Language Leaders]
@FlatBlack722, @spuljko, @atus - is anyone interested in being a language leader for Croatian?

**Comment by atus** [comment]
Yes. I already applied.

**Comment by FlatBlack722** [comment]
Maybe @atus is more suitable for this since I am not really a dev. But I am very much willing to help out.

**Comment by spuljko** [comment]
If atus has applied that is fine by me.

**Comment by synesthesiam** [comment]
@atus Let's get this PR merged and then I'll add you as a language leader :+1:

**Comment by spuljko** [comment]
If there can be multiple language leaders I can join.

**Reply by synesthesiam** [reply]
Thanks @spuljko! Adding you as a language leader now :slightly_smiling_face:

**Comment by synesthesiam** [comment]
@spuljko when you get a chance, could you please look into translating the `HassGetState` sentences/responses?

**Reply by spuljko** [reply]
I am working on it

**Reply by synesthesiam** [reply]
Great, thank you.
